### PR TITLE
Use common base image for services 

### LINKED ISSUE #17: osbs-box.py --help
The current help output for `osbs-box.py` provides minimal information:
```bash
 $ ./osbs-box.py --help
usage: osbs-box.py [-h] [--no-cleanup] [--force-rebuild] {up,down,cleanup}

positional arguments:
  {up,down,cleanup}

optional arguments:
  -h, --help         show this help message and exit
  --no-cleanup
  --force-rebuild
```

Ideally the subcommands shouldn't all be listed as "positional arguments", and a quick description for each should be provided.

### LINKED ISSUE #25: Create a single base image for all services
This would be much easier to update and construct, after PR for multiple distributions is merged

### PR DESCRIPTION
**Pull request by vrutkovs**
Rebase all services on one image, which would accept all build args

Fixes #25

### CODE REVIEW

File: base/Dockerfile.fedora [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+MAINTAINER Luiz Carvalho <lucarval@redhat.com>
+
+ARG UPDATES
+ARG UPDATES_TESTING
+
+RUN if [ $UPDATES ]; then dnf update -y; fi && \
+    if [ $UPDATES_TESTING ]; then \
+            dnf install -y dnf-plugins-core && \
+            dnf config-manager --set-enable updates-testing &&\
+            dnf -y update; fi
```
Comment (lcarva) [inline base/Dockerfile.fedora]: Missing newline at end of file.

File: base/Dockerfile.rhel7 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM rhel7:latest
+MAINTAINER Luiz Carvalho <lucarval@redhat.com>
+
+ARG REPO_URL
+ARG UPDATES
+
+ADD ${REPO_URL} /etc/yum.repos.d/internal.repo
+RUN if [ $UPDATES ]; then yum update -y; fi
```
Comment (lcarva) [inline base/Dockerfile.rhel7]: Missing newline at end of file.

File: base/Dockerfile.fedora (hunk 1 above)
Comment (vrutkovs) [inline base/Dockerfile.fedora]: Fixed

**Comment by vrutkovs** [comment]
>Why only yum refresh in fedora Dockerfiles?

You mean `dnf -y --refresh`? This refreshes metadata only (as base image might be weeks old etc.). There doesn't seem to be an option to do that for yum

**Pull request merged**